indico/modules/events/registration/schemas.py

```diff
@@ -134,10 +134,12 @@ def _get_registration_data(self, registration):
             if field['id'] in filenames:
                 obj = registration.data_by_field[field['id']]
                 field_data['data'] = filenames[field['id']]
-                if field['inputType'] == 'picture':
-                    field_data['data'] = url_for('.registration_picture', obj.locator.file, _external=True)
-                else:
-                    field_data['data'] = filenames[field['id']]
+                picture_data = {
+                    'filename': filenames[field['id']],
```

Comment:
Are we using this in the end? FWIW the only reason why I originally suggested the separate data key is because I thought it'd be cleaner. But if this requires lots of changes in the mobile app then it's not worth it.

So far far as I'm concerned, we can just put the URL in `data` for picture fields...

Reply:
This was my suggestion - I thought keeping the filename around could be useful at some point, but maybe we can get away with just the url